main.py

```diff
@@ -189,17 +202,16 @@ def isAdminLogin(dbSession, username, token):
     return False
 
 
-@app.route('/nachhilfeboerse/api/getSubjects', methods=['GET'])
+@app.route("/nachhilfeboerse/api/getSubjects", methods=["GET"])
 def getSubjects():
 
     dbSession = DB()
-    res = {}
-    for subject in dbSession.query(Subjects).all():
-        res.update({subject.shortcut: subject.long_name})
+    res = {subject.shortcut: subject.long_name for subject in dbSession.query(
+        Subjects).all()}
 
     resp = make_response(jsonify(res))
 
-    resp.headers['Access-Control-Allow-Origin'] = '*'
+    resp.headers["Access-Control-Allow-Origin"] = "*"
     return resp
 
 
```

Comment:
Bei dem TestFlask muss das noch geändert werden
```suggestion
@app.route("/nachhilfeboerse/api/testFlask", methods=["GET"])
def testFlask():
    resp = make_response(jsonify({"status": "working"}))
    resp.headers["Access-Control-Allow-Origin"] = "*"
return resp
```

Reply:
Was genau soll geändert werden?

Reply:
Die Anführungszeichen.
Du hattes das ersetzt, bevor du gepullt hattest